### PR TITLE
Prepare for cran (mainly linting)

### LINKED ISSUE #25: address R CMD CHECK issues

### PR DESCRIPTION
**Pull request by lshandross**

### CODE REVIEW

File: tests/testthat/test-make_d_fn.R [hunk 1]
```diff
@@ -18,178 +18,180 @@
...
+          })
 
 
-test_that("make_d_fn generates error, no continuous component; two point masses, non-zero, duplicated values second only", {
-    ps <- seq(from = 0.3, to = 0.9, by = 0.1)
-    qs <- c(rep(1.0, 1), rep(2.0, 6))
-    expect_error(make_d_fn(ps, qs))
-})
+test_that("make_d_fn generates error, no continuous component;
+two point masses, non-zero, duplicated values second only", {
```
Comment (elray1) [inline tests/testthat/test-make_d_fn.R]:
```suggestion
          two point masses, non-zero, duplicated values second only", {
```

File: tests/testthat/test-make_p_fn_make_q_fn.R [hunk 2]
```diff
@@ -23,649 +23,621 @@
...
-    #                        make_q_fn(ps, qs)(p_actual),
-    #                        tolerance = 1e-3)
-})
+test_that("make_p_fn, make_q_fn work, no continuous component;
+          two point masses, non-zero, duplicated values first only", {
+            ps <- seq(from = 0.1, to = 0.4, by = 0.1)
+            qs <- c(rep(1.0, 3), rep(2.0, 1))
 
+            test_ps <- sort(c(1 / 3, 1.0, seq(from = 0.01, to = 0.99, by = 0.01)))
+            test_qs <- c(1, 2, qnorm(seq(from = 0.01, to = 0.99, by = 0.01), mean = 1, sd = 2))
```
Comment (elray1) [inline tests/testthat/test-make_p_fn_make_q_fn.R]: The indentation situation here is a little funky.  Let's indent to 2 spaces to be consistent with the rest of the stuff.

Comment (elray1) [reply tests/testthat/test-make_p_fn_make_q_fn.R]: continues below

File: tests/testthat/test-make_p_fn_make_q_fn.R [hunk 3]
```diff
@@ -23,649 +23,621 @@
...
-    
-    p_hat <- distfromq::make_p_fn(ps, qs)
-    testthat::expect_identical(p_hat(c(61, Inf)), c(0.99, 1.0))
-})
+            q_hat <- distfromq:::make_q_fn(ps, qs)
+            testthat::expect_identical(q_hat(c(0.99, 1)), c(61, Inf))
+
+            p_hat <- distfromq::make_p_fn(ps, qs)
+            testthat::expect_identical(p_hat(c(61, Inf)), c(0.99, 1.0))
+          })
```
Comment (elray1) [inline tests/testthat/test-make_p_fn_make_q_fn.R]: could fix up the indentation in this part

File: tests/testthat/test-split_disc_cont_ps_qs.R [hunk 4]
```diff
@@ -18,293 +18,296 @@
...
+            expect_equal(
+              split_disc_cont_ps_qs(ps, qs),
+              list(
+                disc_weight = 1.0,
+                disc_ps = c(3 / 9, 6 / 9), disc_qs = c(1.0, 2.0),
+                cont_ps = numeric(), cont_qs = numeric(),
+                disc_ps_range = list(c(0.0, 3 / 9), c(3 / 9, 1.0))
+              )
+            )
+          })
```
Comment (elray1) [inline tests/testthat/test-split_disc_cont_ps_qs.R]: need to fix up indentation here

File: tests/testthat/test-split_disc_cont_ps_qs.R [hunk 5]
```diff
@@ -18,293 +18,296 @@
...
-})
+test_that("split_disc_cont_ps_qs works, two point masses, 
+          is_hurdle with one zero and one non-zero with duplicated qs", {
+            # mixture of a LogNormal(0,1) with weight 0.6,
+            # a point mass at 0 with weight 0.3, and a point mass at 1 with weight 0.1
+
+            # probabilities and quantiles for normal component
+            norm_ps <- seq(from = 0.1, to = 0.9, by = 0.1)
+            norm_qs <- qlnorm(norm_ps)
+            adj_norm_ps <- norm_ps * 0.6 + 0.3 * (norm_qs > 0.0) + 0.1 * (norm_qs > 1.0)
```
Comment (elray1) [inline tests/testthat/test-split_disc_cont_ps_qs.R]: note indentation

**Review by elray1 — CHANGES_REQUESTED** [review]
A few minor places to fix up.

File: tests/testthat/test-make_p_fn_make_q_fn.R (hunk 2 above)
Comment (lshandross) [reply tests/testthat/test-make_p_fn_make_q_fn.R]: The inconsistent indentation here is a result of the first line of the test being 121 characters when the limit is 120 (the other tests generally make the 120 limit). This is the cause of many of the indentation issues you've requested changes for. Would you like me to try and shorten the test names with abbreviations to get around the inconsistent indentation between tests?

Comment (elray1) [reply tests/testthat/test-make_p_fn_make_q_fn.R]: It seems like we should be able to deal with this by inserting some new lines, right?  Here's a different example:

```
test_that("make_d_fn generates error, no continuous component; 
          two point masses, non-zero, duplicated values first only", {
            ps <- seq(from = 0.1, to = 0.4, by = 0.1)
            qs <- c(rep(1.0, 3), rep(2.0, 1))
            expect_error(make_d_fn(ps, qs))
          })
```

which i would think we could update to
```
test_that(
  "make_d_fn generates error, no continuous component; two point masses, non-zero, duplicated values first only",
  {
    ps <- seq(from = 0.1, to = 0.4, by = 0.1)
    qs <- c(rep(1.0, 3), rep(2.0, 1))
    expect_error(make_d_fn(ps, qs))
  }
)
```

Comment (elray1) [reply tests/testthat/test-make_p_fn_make_q_fn.R]: (i'm also up for shortening the test names as needed)

Comment (lshandross) [reply tests/testthat/test-make_p_fn_make_q_fn.R]: Ok, I've gone ahead and implemented your suggested solution with inserting some new lines

**Review by elray1 — APPROVED** [review]

**Pull request merged**